roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -512,7 +505,7 @@ def main():
     ######################################################################
     for os_cert in filter_paths(openshift_cert_check_paths):
         # Open up that config file and locate the cert and CA
-        with open(os_cert, 'r') as fp:
+        with io.open(os_cert, 'r', encoding='utf-8') as fp:
```

Comment:
I would like to get @ashcrow and @rhcarvalho to do a review of the encoding/decoding.